src/pybind/mgr/dashboard/module.py

```diff
@@ -406,6 +411,40 @@ def set_mgr_created_self_signed_cert(self):
             return result
         return 0, 'Self-signed certificate created', ''
 
+    @CLICommand("dashboard get issue")
+    def get_issues_cli(self, issue_number: str):
+        if issue_number == 'None':
+            return -errno.EINVAL, '', 'Invalid arguments. Issue number is missing'
+        cephTrackerClient = CephTrackerClient()
+        try:
+            response = cephTrackerClient.get_issues(issue_number)
+        except RequestException as error:
+            if error.status_code == 404:
+                return errno.EINVAL, '', 'Issue not found'
+            else:
+                return errno.EINVAL, '', 'Unknown error occurred: ' + error
+        return 0, str(response), ''
+
+    @CLICommand("dashboard create issue")
+    def report_issues_cli(self, project: str, tracker: str, subject: str, description: str):
```

Comment:
This is how it is represented on the tracker. But atleast I was referring to project as component, because it is actually a component. So I am good for making this change. 

![Screenshot from 2021-08-23 12-34-38](https://user-images.githubusercontent.com/71764184/130404678-1425f1b3-d8b6-4f38-b002-f50c8dc834b7.png)

Reply:
For now we can go with the current name as  it can be easily updated if needed (I think we are more used to "component" partly because we're ceph developers XD).